anthem/lyrics/loaders.py

```diff
@@ -5,7 +5,7 @@
 import codecs
 import csv
```

Comment:
using unicodecsv would make this easier to maintain, isn't it?

Reply:
indeed!

Reply:
@TDu could you just spend 30 minutes and check if it works with unicodecsv?

Reply:
I have made a new PR based on this one to test with unicodecsv https://github.com/camptocamp/anthem/pull/29